python/lsst/pex/exceptions/wrappers.py

```diff
@@ -1,7 +1,10 @@
+from future import standard_library
+standard_library.install_aliases()
```

Comment:
For my education: How do we know when to do this in our code?  Or is it done by some munging script?